src/main/resources/config.yml

```diff
@@ -265,6 +265,10 @@ Places:
       Message: "&4One thousand placed stones!"
       Name: place_1000_stone
       DisplayName: Stone God
+      Reward:
+        Command:
+          Command: "say §7ALERT: §4PLAYER§7 has become the §4STONE GOD§7!"
```

Comment:
In my opinion having two nested `Command` sections is confusing. Could we rename the new inner-most one to something like `Execute`?

Reply:
Yep, sounds fine.